website/blog/2020-05-06-jest-26.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Jest 26: Tick Tock
+author: Christoph Nakazawa
+authorURL: http://twitter.com/cpojer
+authorFBID: 100000023028168
+---
+
+# Jest 26: Tick Tock
+
+When we started rebuilding Jest five years ago our goal was to provide a batteries-included zero-configuration test runner that is approachable for beginners, extensible for almost all testing use cases and scalable to large projects. One of the most important releases was [Jest 15](https://jestjs.io/blog/2016/09/01/jest-15) which tied all the bits and pieces together and provided great defaults that allowed people to install and run Jest often without any setup. However, this approach has a big downside as Jest installs a lot of dependencies into your projects that you may not need.
+
+We are now beginning to address this shortcoming and are working on reducing Jest’s install size while keeping it approachable and extensible. We have made the following breaking changes in Jest 26:
+
+<!--truncate-->
+
+* `[expect, jest-mock, pretty-format]` Remove `ES5` build files with a new minimum of support of ES2015 (Node 8) which were only used for browser builds ([#9945](https://github.com/facebook/jest/pull/9945))
```

Comment:
Will jest simply drop support for ES5 environments or is there some migration path available? That path should probably be mentioned in the blog post.

Reply:
the plan is https://github.com/facebook/jest/pull/9635#issuecomment-623093145, we haven't done it yet though